week3/conf/ltr_featureset.json

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
Wow, looks like you're already pretty advanced! Good one!

Reply:
These are some changes pushed by **gsingers** to the main repo..My pull request has incorporated those too